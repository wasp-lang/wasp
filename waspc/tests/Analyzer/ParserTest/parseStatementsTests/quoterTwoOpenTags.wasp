test TwoOpenTags {=foo {=foo foo=}
