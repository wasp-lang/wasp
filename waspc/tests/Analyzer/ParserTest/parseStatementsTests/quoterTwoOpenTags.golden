(AST
  (Decl@1:1-34 type=test name=TwoOpenTags
    (Quoter@1:18-34 tag=foo
      {= {=foo =}
    )
  )
)
